var/spack/repos/builtin/packages/ceed/package.py

```diff
@@ -125,6 +152,9 @@ class Ceed(BundlePackage):
     depends_on('hpgmg@a0a5510df23b+fe', when='@1.0.0+petsc')
 
     # MAGMA
+    # ceed-4.0
+    depends_on('magma@2.5.4', when='@4.0.0+cuda')
+    depends_on('magma@2.5.4+hip', when='@4.0.0+hip')
```

Comment:
Currently the magma spack package does not have `hip` variant.

@stomov, @nbeams, is adding a `hip` variant for magma 2.5.4 possible at all? If so, is it relatively easy to add here?

Reply:
ping: @stomov, @nbeams

Reply:
There are not currently plans to add a `hip` spack variant for MAGMA 2.5.4, but there will be for the upcoming release of MAGMA 2.6.  Stan says he can hopefully work on the CMake and spack packages for 2.6 this week.

Reply:
Okay, thanks for the update. Since we (most probably) don't want to wait for magma v2.6, I'll update the `ceed` package to remove comments about `magma+hip`. We can postpone the `magma+hip` support in `ceed` to the next CEED release.